src/lib/http.coffee

```diff
@@ -21,11 +23,30 @@ module.exports = (dnschain) ->
             @log = gNewLogger 'HTTP'
             @log.debug "Loading HTTPServer..."
             @rateLimiting = gConf.get 'rateLimiting:http'
+            app = express()
+            router = express.Router()
+
+            router.route(/\/v1\/(\w+)\/(\w+)(\/([^\/]+)(\/(\w+)(\.(\w+))?)?)?/)
```

Comment:
Did you try making this clearer using the express syntax?

```coffeescript
router.route("/v1/:resolver([^\\/]+?)/:resource ... etc
```

Not sure if that regex is correct, RTFM to see what the correct syntax is.